lib/java/logging-service-opensearch/src/main/java/org/enso/logging/service/opensearch/OpenSearchAppender.java

```diff
@@ -0,0 +1,33 @@
+package org.enso.logging.service.opensearch;
+
+import java.net.URI;
+import java.util.concurrent.ThreadPoolExecutor;
+import org.enso.logging.service.AuthenticationData;
+import org.enso.logging.service.LogJobsProcessor;
+import org.enso.logging.service.TokenRefresher;
+import org.slf4j.LoggerFactory;
+
+@org.openide.util.lookup.ServiceProvider(
+    service = org.enso.logging.service.logback.RemoteAppender.class)
+public final class OpenSearchAppender extends org.enso.logging.service.RemoteAppender {
```

Comment:
Not sure what witticism to use? Maybe DRY? Why do we thing it make sense to **have three classes of the same name** in the system?

![obrazek](https://github.com/user-attachments/assets/5603245c-b256-468d-934a-ae6a3a2e389b)

This kind of _use the same name three times for three different things_ confuses me every time I try to approach this part of code base.

Reply:
The separation allows us to not tightly couple with a particular appender in runtime deps. This is following the same pattern as [Telemetry Appender PR](#12488) and you didn't voice concerns about it back then.